version.php

```diff
@@ -36,10 +36,10 @@
 
 defined('MOODLE_INTERNAL') || die;
 
-$plugin->version    = 2016052401;
+$plugin->version    = 2016112701;
 $plugin->component  = 'mod_openmeetings';
 $plugin->maturity   = MATURITY_STABLE;
-$plugin->requires   = 2014051200;
-$plugin->release    = '2.0.0.4';
+$plugin->requires   = 2016112500;
```

Comment:
OK, But since PHP code was not changed, plugin would work with 
Moodle 3.0, PHP 5
as well as with 
Moodle 3.2, PHP 7

I see no need to change `requires` field and drop 3.0 support ....